collection-scripts/gather_noobaa_resources

```diff
@@ -42,6 +47,31 @@ mkdir -p "${NOOBAA_COLLLECTION_PATH}/raw_output/}"
 dbglog "Collecting MCG database information..."
 oc rsh --namespace "${INSTALL_NAMESPACE}" noobaa-db-pg-0 psql -d nbcore -c '\pset pager off' -c '\list+' -c '\dt+' &>"${NOOBAA_COLLLECTION_PATH}"/raw_output/db_list.txt 2>&1
 
+dbglog "Collecting CNPG cluster information..."
+mkdir -p "${NOOBAA_COLLLECTION_PATH}/cnpg_info/"
+# Get the status of the CNPG cluster
+oc cnpg -n "${INSTALL_NAMESPACE}" status noobaa-db-pg-cluster -vv &>"${NOOBAA_COLLLECTION_PATH}"/cnpg_info/cnpg_cluster_status.txt 2>&1
```

Comment:
`oc cnpg` is not a valid command? I think you meant to call the copied `kubectl-cnpg` binary? If so, use `oc rsh`.

Also, where is this requirement coming from? It is not a 1st party tool. Do we really need this tool?

Reply:
The CNPG plugin is a Kubernetes kubectl plugin (see [here](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/)). Once it is placed in the `PATH`, it is possible to use it by calling `kubectl [plugin name]`, and it also works with `oc`, of course.
This tool is part of the cnpg project we are using for noobaa HA. We build a slightly modified version of this tool together with the operator itself, and the binary is provided inside the cnpg-operator image for support purposes (see [here](https://issues.redhat.com/browse/RHSTOR-6814))

Reply:
Do we want to continue in cases where the plugin is not copied to the pod? If yes, LGTM else please add a check for that.

Reply:
What do you mean by not copied to the pod? If it fails in `cp_cnpg_cli`? We do want to continue and not break the rest of the info gathering.